src/webui/api/serialize/serialize_torrent.cpp

```diff
@@ -171,6 +171,7 @@ QVariantMap serialize(const BitTorrent::Torrent &torrent)
         {KEY_TORRENT_POPULARITY, torrent.popularity()},
         {KEY_TORRENT_SEEDING_TIME_LIMIT, torrent.seedingTimeLimit()},
         {KEY_TORRENT_INACTIVE_SEEDING_TIME_LIMIT, torrent.inactiveSeedingTimeLimit()},
+        {KEY_TORRENT_SHARE_LIMIT_ACTION, static_cast<int>(torrent.shareLimitAction())},
```

Comment:
I strongly dislike enums encoded as numbers.
You can search for `Utils::String::fromEnum()` usages in the code to see how encode it to string.

Reply:
fixed in https://github.com/qbittorrent/qBittorrent/pull/22989/commits/fccbd816a9b82e65fdea6521abe58e0d7d7211f5